addons/website/models/res_users.py

```diff
@@ -21,7 +21,6 @@ class ResUsers(models.Model):
     @api.constrains('login', 'website_id')
     def _check_login(self):
```

Comment:
Do it work today?
`_login_key` does already the job. Maybe review its definition.